Task 2/2.2/Moveable.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace _2._2
+{
+    public abstract class Moveable : Objects
```

Comment:
Moveable sounds like an interface for inanimate objects. Call it `Character` (safe option) or `Creature` or something else